frontend/src/components/AssignmentModal/ContactInfoCard.vue

```diff
@@ -0,0 +1,418 @@
+<template>
+  <v-card-text>
+    <v-container 
+      v-for="contactData in assignmentState.contactsInfo"
+      :key="contactData.id"
+    >
+      <v-row v-if="!contactData.id">
+        <v-col
+          cols="12"
+        >
+          <p class="text-subtitle-1">{{ contactData.type }}</p>
+        </v-col>
+      </v-row>
+      <v-row v-else>
+        <v-col
+          cols="12"
+          sm="11"
+        >
+          <CustomSelect 
+            v-model="contactData.type"
+            :items="getContactTypesLabels"
+            :label="'Contact type*'"
+          />
+        </v-col>
+        <v-col
+          v-if="contactData.id"
+          cols="12"
+          sm="1"
+        >
+          <v-btn
+            style="transform: translateY(50%);"
+            color="error"
+            density="compact"
+            icon="mdi-close"
+            @click="() => openConfirmationModal(contactData.id, 'contact')"
+          />
+        </v-col>
+      </v-row>
+
+      <v-row>
+        <v-col
+          cols="12"
+          sm="6"
+        >
+          <CustomInput
+            v-model="contactData.firstName"
+            :rules="[rules.required]"
+            :is-required="true"
+            :label="'First name*'"
+          />
+        </v-col>
+        <v-col
+          cols="12"
+          sm="6"
+        >
+          <CustomInput
+            v-model="contactData.lastName"
+            :rules="[rules.required]"
+            :is-required="true"
+            :label="'Last name*'"
+          />
+        </v-col>
+      </v-row>
+
+      <v-row class="pb-5">
+        <v-col cols="12">
+          <CustomInput
+            v-model="contactData.email"
+            :rules="[rules.required, rules.email]"
+            :is-required="true"
+            :label="'Email*'"
+          />
+        </v-col>
+      </v-row>
+      <v-divider />
+
+      <v-row
+        v-for="phoneData in contactData.phoneNumbers"
+        :key="phoneData.id"
+        :class="phoneData.id ? '' : 'pt-7'"
+      >
+        <v-col 
+          cols="12"
+          sm="4"
+        >
+          <CustomSelect
+            v-model="phoneData.type"
+            :items="phoneNumberTypesLabels"
+            :label="'Phone number type*'"
+          />
+        </v-col>
+        <v-col 
+          cols="12"
+          :sm="phoneData.id ? 7 : 8"
+        >
+          <CustomInput
+            v-model="phoneData.number"
+            :rules="[rules.required, rules.phoneNumber]"
+            :is-required="true"
+            :label="'Phone number*'"
+            :placeholder="'+X XXX-XXX-XXX'"
+          />
+        </v-col>
+        <v-col
+          v-if="phoneData.id"
+          cols="12"
+          sm="1"
+        >
+          <v-btn
+            style="transform: translateY(50%);"
+            color="error"
+            density="compact"
+            icon="mdi-close"
+            @click="() => openConfirmationModal(contactData.id, 'phoneNumber', phoneData.id)"

```

Comment:
Probably we can remove the arrow function here and just use openConfirmationModal(contactData.id, 'phoneNumber', phoneData.id)